### DISCUSSION TITLE
Deploy Discussions to Organizations

### DISCUSSION THREAD

**Discussion by JonathanAsbury-ACS** [community/community#3461] [Discussions]
Based on what I've read across various post and your road map, it appears that Discussions are deployed to a repository, initially public but more recently a handful of private repos, like I requested [here](https://github.com/github/feedback/discussions/2853#discussioncomment-419025).  To date, the term _enable_ has been used in the request but I prefer deploy for reasons I'll describe below.  

Ideally, it would be great if Discussions could be deployed to an organization.  Repos across the organization, whether public or private, could then enable/disable as needed, just like other Tabs such as Actions.  

Currently we have a handful of repos that will us it, but I anticipate it becoming a standard across the organization and feel like we'll end up bombarding you with requests once it catches on.  

Great product!  Our primary use is to facilitate debates about product enhancements that lack clarity to justify being added to the backlog.

**Comment by carwyn** [comment]
There is more detail on this in #41
